includes/class-wc-payments.php

```diff
@@ -310,9 +309,11 @@ public static function init() {
 			new WC_Payments_Admin_Sections_Overwrite( self::get_account_service() );
 		}
 
-		// Load subscriptions (Stripe Billing).
-		include_once WCPAY_ABSPATH . '/includes/subscriptions/class-wc-payments-subscriptions.php';
-		WC_Payments_Subscriptions::init( self::$api_client, self::$customer_service );
+		// Load WCPay Subscriptions.
+		if ( WC_Payments_Features::is_wcpay_subscriptions_enabled() ) {
+			include_once WCPAY_ABSPATH . '/includes/subscriptions/class-wc-payments-subscriptions.php';
```

Comment:
@mattallan should we put a try/catch around this to gracefully fail when the subs core lib is not correctly in place? (I just hit this - I failed to include it in my test build)